wsapi.go

```diff
@@ -411,10 +411,35 @@ type requestGuildMembersOp struct {
 
 // RequestGuildMembers requests guild members from the gateway
 // The gateway responds with GuildMembersChunk events
-// guildID  : The ID of the guild to request members of
+// guildID  : Single Guild ID to request members of
 // query    : String that username starts with, leave empty to return all members
 // limit    : Max number of items to return, or 0 to request all members matched
-func (s *Session) RequestGuildMembers(guildID, query string, limit int) (err error) {
+func (s *Session) RequestGuildMembers(guildID string, query string, limit int) (err error) {
+	data := requestGuildMembersData{
+		GuildIDs: []string{guildID},
+		Query:    query,
+		Limit:    limit,
+	}
+	err = s.requestGuildMember(data)
+	return err
+}
+
+// RequestGuildMembersBatch requests guild members from the gateway
+// The gateway responds with GuildMembersChunk events
+// guildID  : Slice of guild IDs to request members of
+// query    : String that username starts with, leave empty to return all members
+// limit    : Max number of items to return, or 0 to request all members matched
+func (s *Session) RequestGuildMembersBatch(guildIDs []string, query string, limit int) (err error) {
+	data := requestGuildMembersData{
+		GuildIDs: guildIDs,
+		Query:    query,
+		Limit:    limit,
+	}
+	err = s.requestGuildMember(data)
+	return err
+}
+
+func (s *Session) requestGuildMember(data requestGuildMembersData) (err error) {
```

Comment:
`requestGuildMember` seems to be a typo here; `requestGuildMembers` would make more sense.